modules/quill/4.3.0/patches/add_build_file.patch

```diff
@@ -0,0 +1,19 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,16 @@
++package(default_visibility = ["//visibility:public"],)
++
++licenses(["notice"]) # MIT
++
++cc_library(name = "quill",
++           hdrs = glob(["quill/include/**/*.h"]),
++           includes = ["quill/include"],
++           linkopts = select({
++               "@platforms//os:windows": [],
++               "@platforms//os:linux": ["-lpthread", "-lrt",],
++               "//conditions:default": ["-lpthread",]
++           }),
++           copts = select({
++               "@platforms//os:windows": [],
++               "//conditions:default": ["Wno-gnu-zero-variadic-macro-arguments"],
```

Comment:
This looks like a `gcc`-specific argument. You could select on `@rules_cc//cc/compiler:gcc` instead.